### ISSUE TITLE
rmse = cost有什么用呢？

### ISSUE THREAD

**Issue by AtlaskD** [NeWnIx5991/AutoRec-for-CF#1]
`rmse = cost`之后，rmse和cost还是指向同一个torch，所以不是很懂这一步是在做什么。如果rmse是要记录不带regularization的loss的话，可以改成`rmse = cost.clone().detach()`。
